tools/report-converter/codechecker_report_converter/report/output/html/html.py

```diff
@@ -171,7 +173,9 @@ def _add_source_file(self, file: File) -> FileSource:
             file_content = InvalidFileContentMsg
 
         self.files[file.id] = {
-            'id': file.id, 'filePath': file.path, 'content': file_content}
+            'id': file.id, 'filePath': file.path,
+            'content': replace_script_tag.sub(r"&lt;/\1&gt;", file_content)
```

Comment:
Hi,
Thanks for the patch, this is a very nice catch.
Python has a module named `html` which provides escaping functionality for this situation. Would you think, this could be a more general solution to this issue?
```suggestion
            'content': html.escape(file_content)
```
Of course `html` needs to be imported instead of `re`;

Thanks again!